src/erlcloud_cloudwatch.erl

```diff
@@ -0,0 +1,266 @@
+-module(erlcloud_cloudwatch).
```

Comment:
yes, let's rename it to `erlcloud_cloudwatch_logs` or may be better `erlcloud_cw_logs`